src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -0,0 +1,80 @@
+import logging
+import os
+import torch
+from torch._dynamo.backends.common import fake_tensor_unsupported
+from torch._dynamo.backends.registry import register_backend
+from torch._inductor.compile_fx import compile_fx
+
+from openvino.frontend import FrontEndManager
+from openvino.runtime import Core, Type, PartialShape
+from openvino.frontend.pytorch.decoder import TorchScriptPythonDecoder
+
+log = logging.getLogger(__name__)
+
+'''
+    This is a preview feature in OpenVINO. Torchscript backend
+    enables users to compile PyTorch models using torch.compile
+    with OpenVINO as a target backend.
+
+    Sample usage:
+    This sample code loads resnet50 torchvision model and compiles it using torch dynamo.
+    We can then use this model for inference
+
+    model = torchvision.models.resnet50()
```

Comment:
@suryasidd, it will be better to mention that only two additional lines of code are needed and also add the import statement.